DDG4/python/DDSim/Helper/ConfigHelper.py

```diff
@@ -53,10 +53,10 @@ def __repr__(self):
     return self.printOptions()
 
   def printOptions(self):
-    """print all paramters"""
+    """print all parameters"""
     options = []
     for opt, val in six.iteritems(self.getOptions()):
-      options.append("\n\t'%s': '%s'" % (opt, val[0]))
+      options.append("\n\t'%s': '%s'" % (opt, val['default']))
```

Comment:
This is actually a bug fix, since before you probably got `KeyError`

Reply:
mentioned this in the release notes after creation of PR